module/VuFind/src/VuFind/Controller/AbstractSearch.php

```diff
@@ -449,9 +447,49 @@ protected function processJumpTo($results)
 
         // If we got this far, we have a valid parameter so we should redirect
         // and report success:
-        $details = $this->getRecordRouter()
-            ->getTabRouteDetails($recordList[$jumpto - 1]);
-        return $this->redirect()->toRoute($details['route'], $details['params']);
+        return $this->getRedirectForRecord($recordList[$jumpto - 1]);
```

Comment:
I guess we could now further collapse the end of this function down to:

```
return isset($recordList[$jumpto - 1])
    ? $this->getRedirectForRecord($recordList[$jumpto - 1]) : false;
```

...if we wanted to be a little more concise.

Reply:
Done.